### PR TITLE
migration nickname and rank

### PR DESCRIPTION
**Pull request by Khalidir**

### CODE REVIEW

**Comment by vladiangelov** [comment]
Please add rank and also add these to the controllers and views pages related to users (new/create/edit/etc).

**Comment by brycole** [comment]
rank is missing @Khalidir

File: db/migrate/20200222165406_add_chef_to_users.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddChefToUsers < ActiveRecord::Migration[5.2]
```
Comment (brycole) [inline db/migrate/20200222165406_add_chef_to_users.rb]: please change this migration. it should not be called "add chef to users" @Khalidir

Comment (Khalidir) [reply db/migrate/20200222165406_add_chef_to_users.rb]: the name of the migration was changed

**Comment by brycole** [comment]
please merge master into this branch

File: db/migrate/20200222112653_add_chef_to_user.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddChefToUser < ActiveRecord::Migration[5.2]
```
Comment (brycole) [inline db/migrate/20200222112653_add_chef_to_user.rb]: please destroy this migrate

**Comment by brycole** [comment]
looks good

**Pull request merged**